plugins/cargo/cargo.plugin.sh

```diff
@@ -0,0 +1,19 @@
+#! bash oh-my-bash.module
+# @sjplanet 2025
+
+# Set CARGO_HOME to default if not already set
+: "${CARGO_HOME:=$HOME/.cargo}"
+
+# Check if the directory exists
+if [[ -d "$CARGO_HOME" ]]; then
+  if [[ -f "$CARGO_HOME/env" && -r "$CARGO_HOME/env" ]]; then
```

Comment:
`[[ ... ]]` is not a normal command, but a special syntax of Bash that was introduced to solve the quoting issues with `[ ... ]`. One of the issues is the word splitting and the pathname expansions that are applied to the arguments of the normal commands such as `[`. To prevent the word splitting and the pathname expansions, users needed to quote the arguments as `"$var"`. `[[ ... ]]` was introduced to solve such an issue so that the users do not need to quote the expansions as `"$var"`. Thus, quoting parameter expansions in `[[ ... ]]` does not make sense.

Reply:
In fact, [the page you provided](https://guide.bash.academy/expansions/?=Command_Substitution#a1.3.0_1) includes the following example:

```bash
[[ $- = *i* ]]
```

while there are no examples of quoting the parameter expansion of `[[ ... ]]`.

Reply:
I'll drop those commits since there are no replies from the OP, I'm pretty sure these are not problems, and these are inconsistent with the existing styles of the codebase.